### PR TITLE
Added support for SPEEDYBEEF4 board

### PR DESCRIPTION
**Pull request by dbasch**
Things tested:

- GYRO/ACC working properly, right orientation
- UART 3 works with a gps
- UART 2 works with an sbus receiver
- MOTOR 1 works
- SERVOS 3 and 4 work
- onboard flash works with blackbox
- bluetooth works
- OSD works

Untested: i2c, other motor outputs, ppm

Flew a plane for about 30 minutes total, no issues.

### CODE REVIEW

**Review by shellixyz — CHANGES_REQUESTED** [review]
Why is your PR including stuff in other PRs that are not been merged yet ? Please only include the files contained in `target/SPEEDYBEEF4` in your PR.

**Comment by dbasch** [comment]
Just did a clean fork and added the changes. Is this better?

**Comment by shellixyz** [comment]
Better but there are still changes in these files that shouldn't belong in this PR:
- src/main/navigation/navigation_private.h
- src/main/drivers/flash_m25p16.c (debug stuff)

Also please fix the indentation in the mk file

**Comment by dbasch** [comment]
Hopefully fixed. BTW the .mk file was copied from the OMNIBUSF4 one that seems to have weird indentation.

File: src/main/drivers/flash_m25p16.c [hunk 1]
```diff
@@ -17,6 +17,7 @@
 
 #include <stdbool.h>
 #include <stdint.h>
+#include "build/debug.h"
```
Comment (shellixyz) [inline src/main/drivers/flash_m25p16.c]: Please remove this line

File: src/main/drivers/flash_m25p16.c [hunk 2]
```diff
@@ -150,10 +151,12 @@ static bool m25p16_readIdentification(void)
     chipID = (in[1] << 16) | (in[2] << 8) | (in[3]);
 
     // All supported chips use the same pagesize of 256 bytes
+    DEBUG_TRACE("%x\n", chipID);
```
Comment (shellixyz) [inline src/main/drivers/flash_m25p16.c]: Please remove this line

**Review by shellixyz — CHANGES_REQUESTED** [review]
Please remove the two lines. Otherwise seems all good ! Thanks

File: src/main/drivers/flash_m25p16.c (hunk 1 above)
Comment (dbasch) [reply src/main/drivers/flash_m25p16.c]: done

File: src/main/drivers/flash_m25p16.c (hunk 2 above)
Comment (dbasch) [reply src/main/drivers/flash_m25p16.c]: done

**Review by shellixyz — APPROVED** [review]

**Review by digitalentity — APPROVED** [review]

**Pull request merged**